OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -30,15 +30,17 @@ public class Shroud
 		readonly Map map;
 
 		readonly CellLayer<short> visibleCount;
-		readonly CellLayer<short> generatedShroudCount;
 		readonly CellLayer<bool> explored;
+		readonly CellLayer<short> generatedDisruptionCount;
+		readonly CellLayer<short> generatedShroudCount;
```

Comment:
It doesn't work when I make this CellLayer<ushort>. Why? Does it ever fall to negative values?